File: backend/src/main/java/com/gestion/blanchiment/dto/AgenceDto.java
package com.gestion.blanchiment.dto;

public class AgenceDto {
	private Integer id_pays;
	private String liblon;
	private String libcou;
	private String adresse;
	private String code_interne;
	
	public Integer getId_pays() {
		return id_pays;
	}
	public void setId_pays(Integer id_pays) {
		this.id_pays = id_pays;
	}
	public String getLiblon() {
		return liblon;
	}
	public void setLiblon(String liblon) {
		this.liblon = liblon;
	}
	public String getLibcou() {
		return libcou;
	}
	public void setLibcou(String libcou) {
		this.libcou = libcou;
	}
	public String getAdresse() {
		return adresse;
	}
	public void setAdresse(String adresse) {
		this.adresse = adresse;
	}
	public String getCode_interne() {
		return code_interne;
	}
	public void setCode_interne(String code_interne) {
		this.code_interne = code_interne;
	}
	public AgenceDto(Integer id_pays, String liblon, String libcou, String adresse, String code_interne) {
		super();
		this.id_pays = id_pays;
		this.liblon = liblon;
		this.libcou = libcou;
		this.adresse = adresse;
		this.code_interne = code_interne;
	}
	
	
}

File: frontend/src/app/Model/agencedto.ts
export class AgenceDto{

    private libcou :string;
     private liblon :string;
     private adresse :string;
     private id_pays:number;
     private code_interne:string;

  
    public constructor(libcou:string,liblon:string,adresse:string,code_interne:string,id_pays:number){
     this.libcou=libcou
     this.liblon=liblon
     this.adresse=adresse
     this.code_interne=code_interne
     this.id_pays=id_pays    
    }
  }